packages/account-v2/src/containers/POAForm/DocumentSubmission/DocumentSubmission.tsx

```diff
@@ -63,7 +63,7 @@ const DocumentSubmission: React.FC = () => {
                         descriptionSize='2xs'
                         fileFormats={['image/jpeg', 'image/jpg', 'image/png', 'image/gif', 'application/pdf']}
                         hoverMessage='Upload your file here'
-                        icon={<StandaloneArrowUpFromBracketBoldIcon iconSize='lg' />}
+                        icon={<StandaloneArrowUpFromBracketBoldIcon fill='#C7E5E5' iconSize='lg' />}
```

Comment:
you can use define the color in tailwind config and use the tailwind fill to apply the color.
https://tailwindcss.com/docs/fill

Reply:
giving a `className` will lead to applying that class to the entire icon container. also, passing the `tailwind class` as fill prop too is not working.